sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -345,6 +346,47 @@ class CSVSuite extends QueryTest with SharedSQLContext with SQLTestUtils with Te
     assert(result.schema.fieldNames.size === 1)
   }
 
+  test("SPARK-26339 Debug statement if some of specified paths are filtered out") {
+    class TestAppender extends AppenderSkeleton {
+      var events = new java.util.ArrayList[LoggingEvent]
+      override def close(): Unit = {}
+      override def requiresLayout: Boolean = false
+      protected def append(event: LoggingEvent): Unit = events.add(event)
+    }
+
+    val testAppender1 = new TestAppender
+    val rootLogger = LogManager.getRootLogger
+    val origLevel = rootLogger.getLevel
+    rootLogger.setLevel(Level.DEBUG)
+    rootLogger.addAppender(testAppender1)
+    try {
+      val cars = spark
+        .read
+        .format("csv")
```

Comment:
Nit: you can use `.csv` instead of `format` and `load`

Reply:
I fixed it.